docs/developer-guide/documentation.md

```
@@ -26,7 +26,7 @@ so that they remain relevant. Open a commit adding it to our `docs/design` folde
 
 * **Feature Docs**: If your enhancement proposal has merged; next step is to
 implement that feature. As part of the main implementation PR we mandate adding a
-feature documentation commit. See [here](https://github.com/ovn-org/ovn-kubernetes/blob/master/docs/features/template.md)
+feature documentation commit. See [here](https://github.com/ovn-kubernetes/ovn-kubernetes/blob/master/docs/features/template.md)
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)